config/settings.yml

```diff
@@ -1051,6 +1051,11 @@
       :starting_timeout: 20.minutes
       :poll: 10.seconds
       :memory_threshold: 0.megabytes
+    :amazon_agent_manager_worker:
+      :starting_timeout: 20.minutes
+      :heartbeat_timeout: 30.minutes
+      :poll: 10.seconds
```

Comment:
Is this too frequent...this implies that every 10 seconds we're going to check for the request/reply queue, which might be a little expensive?